server/src/main/java/org/elasticsearch/action/admin/cluster/configuration/AddVotingConfigExclusionsRequest.java

```diff
@@ -39,6 +43,8 @@
  */
 public class AddVotingConfigExclusionsRequest extends MasterNodeRequest<AddVotingConfigExclusionsRequest> {
     private final String[] nodeDescriptions;
+    private String[] nodeIds = null;
```

Comment:
Before when I declared `private final String[] nodeId;` instead of `private String[] nodeIds = null;`, I got an IDE compiler warning `Variables might not have been initialized`. The reason was due to the constructor initializing these fields using StreamInput only initialize them for new version. I've made an update to the constructor to allow this declaration.